packages/backend/src/server/api/endpoints/admin/show-users.ts

```diff
@@ -98,13 +98,13 @@ export default define(meta, paramDef, async (ps, me) => {
 
 	switch (ps.origin) {
 		case "local":
-			query.andWhere("user.host IS NULL AND user.username NOT LIKE '%_at_tumblr_com'");
```

Comment:
How much do we care that a mischief-minded person could sign up with a username that ends in `_at_tumblr_com`?

Reply:
nah, it's not that it could do a lot of harm: worst-case scenario would be they would be excluded from some timelines, etc, so ... also, the admins could just spot them and shot it down if they want